<launch>
    <node pkg="competition" name="competition_controller" type="competition_node" output="screen" >
        <rosparam file="$(find competition)/config/mechatronics_config.yaml" />
    </node>
</launch>